### DISCUSSION TITLE
facing problems on lammps coding

### DISCUSSION THREAD

**Discussion by hasan-du-1986** [community/community#54856] [Programming Help]
### Hi everyone,

Dear All,
 I am a new user of lammps. i am creating my lammps data files using moltemplate. I am working on isobutane (R600a) and using COMPASS force field using moltemplate. the code for isobutane generated by moltemplate running smoothly but the out put results is not satisfactory. My target is to cooling the isobutane from superheat region, that is vapor to liquid isobutane.  Temperature is coming close to the targeted value. But pressure is fluctuating so much and also the enthalpy is very low. 
What should I do? Seeking advise from exparts.

Thanks in advance.
Thanks
Rakibul Hasan

![image](https://user-images.githubusercontent.com/83562116/236814581-2cceb9bc-8252-4576-83b1-a95f65877133.png)
![image](https://user-images.githubusercontent.com/83562116/236816265-2e209412-51bc-49b1-a679-366d9d1a907c.png)
![image](https://user-images.githubusercontent.com/83562116/236816304-3469866a-d832-49f0-91b3-3ce1b141c4b6.png)
![image](https://user-images.githubusercontent.com/83562116/236816347-f023732d-dcd3-4e50-bd9d-335cffcc7169.png)
![image](https://user-images.githubusercontent.com/83562116/236816382-89394775-9689-44d2-99dc-d895455756c9.png)
![image](https://user-images.githubusercontent.com/83562116/236816437-713ed8f3-10cb-4f06-b705-cf56af766acd.png)

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @hasan-du-1986 , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!
